fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordCursor.java

```diff
@@ -190,7 +190,13 @@ default boolean hasNext() {
          * @see ExecuteProperties.Builder#setScannedRecordsLimit(int)
          * @see RecordScanLimiter
          */
-        SCAN_LIMIT_REACHED(true);
+        SCAN_LIMIT_REACHED(true),
+
+        /**
+         * The limit on the number of bytes to scan was reached.
+         * {@link #getContinuation()} may return a continuation for resuming the scan.
```

Comment:
For parallelism with the other docs, this should probably include `@see` directives to the `ExectueProperties.Builder` method and the `ByteScanLimiter`.

Reply:
Added.